angular-client/src/pages/graph-page/graph-page.component.html

```diff
@@ -9,4 +9,7 @@
 <ng-template #displayData>
   <graph-header />
   <sidebar [nodes]="nodes!" />
+  <div style="position: absolute; left: 100px">
```

Comment:
why is this absolute positioning, also this should be on the landing page not the graph page

Reply:
this was just placeholder while i was making it, i'll move it to landing page